CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/CVKSwapChain.java

```diff
@@ -416,16 +402,18 @@ private int InitVKFrameBuffer(MemoryStack stack) {
         
         framebufferHandles = new ArrayList<>(imageCount);
 
-        for (long imageView : imageViewHandles) {
-            attachments.put(0, imageView);
+        swapChainImages.forEach(image->{
+            attachments.put(0, image.GetImageViewHandle());
             attachments.put(1, cvkDepthImage.GetImageViewHandle());
             framebufferInfo.pAttachments(attachments);
-            ret = vkCreateFramebuffer(cvkDevice.GetDevice(), 
+            int retVal = vkCreateFramebuffer(cvkDevice.GetDevice(), 
                                       framebufferInfo, 
                                       null, //allocation callbacks
                                       pFramebuffer);
-            framebufferHandles.add(pFramebuffer.get(0));
-        }
+            checkVKret(retVal);
```

Comment:
we should return if something went wrong.  The checkVKret should be inside constructors or void functions we can't change.

Reply:
I can rewrite it as a normal for loop (can't return a value from a lambda)?

Reply:
Yeah the lambdas kinda suck, you can't return from them and the debugger jumps them in a single step.  Please change to a for (var image : swapChainImages)

Reply:
Changed to:
for (var image : swapChainImages) {
            attachments.put(0, image.GetImageViewHandle());
            attachments.put(1, cvkDepthImage.GetImageViewHandle());
            framebufferInfo.pAttachments(attachments);
            ret = vkCreateFramebuffer(cvkDevice.GetDevice(), 
                                      framebufferInfo, 
                                      null, //allocation callbacks
                                      pFramebuffer);
            if (VkFailed(ret)) { return ret; }
            framebufferHandles.add(pFramebuffer.get(0)); 
        }